e2e/cluster.go

```diff
@@ -339,6 +339,24 @@ func (c *Cluster) startNode(name string) {
 	c.nodes[name].Start()
 }
 
+// MaxFaulty is a wrapper function which invokes MaxFaultyVotingPower on PBFT consensus instance of the first node in cluster
+func (c *Cluster) MaxFaulty() (uint64, error) {
+	nodes := c.getNodes()
+	if len(nodes) == 0 {
+		return 0, errors.New("unable to determine max faulty nodes: cluster is empty")
+	}
+	return nodes[0].pbft.MaxFaultyVotingPower(), nil
+}
+
+// QuorumSize is a wrapper function which invokes QuorumSize on PBFT consensus instance of the first node in cluster
+func (c *Cluster) QuorumSize() (uint64, error) {
+	nodes := c.getNodes()
+	if len(nodes) == 0 {
+		return 0, errors.New("unable to determine max faulty nodes: cluster is empty")
```

Comment:
Let's pre-define this error on the package level

Reply:
Oh, I've just noticed that this error message isn't correct (copy paste error).
It should be like this:
```go
errors.New("unable to determine max faulty nodes, because cluster is empty")
```
Do you still find beneficial to extract it to some package level variable?